modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -1,23 +1,34 @@
 // Module included in the following assemblies:
 //
-// * ingress/configure-ingress-operator.adoc
+// * ingress/ingress-operator.adoc
```

Comment:
Should this be `networking/ingress-operator.adoc`?

Reply:
Correct. Updated.

![Screenshot from 2024-06-28 11-38-22](https://github.com/openshift/openshift-docs/assets/57954076/3d0effbe-692b-4343-8858-94f5e4f56745)